src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/Extensions/SAP_Internal_Stats_KeyFigure+Formatting.swift

```diff
@@ -29,6 +28,14 @@ extension SAP_Internal_Stats_KeyFigure {
 		}
 	}
 
+	var formattedValueWithPercent: String? {
+		let numberFormatter = NumberFormatter()
+		numberFormatter.numberStyle = .percent
+		let decimals = max(0, Int(self.decimals))
+		numberFormatter.maximumFractionDigits = Int(decimals)
+		return numberFormatter.string(from: NSNumber(value: value / 100))?.filter({ !$0.isWhitespace })
```

Comment:
Is the value that we get not already a percent value? Do we need to divide it by 100?

Reply:
no the Formatted deals with the value of 1 as 100%  so if you pass 4 as a number it is considered 400% that is why i divide first